File: api-posts/src/main/java/com/github/alexeysol/geekregime/apiposts/configurations/JpaConfiguration.java
package com.github.alexeysol.geekregime.apiposts.configurations;

import com.github.alexeysol.geekregime.apicommons.utils.search.SearchableRepositoryImpl;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EnableJpaRepositories(
    basePackages = "com.github.alexeysol.geekregime.apiposts.repositories",
    repositoryBaseClass = SearchableRepositoryImpl.class
)
public class JpaConfiguration {}

File: api-users/src/main/kotlin/com/github/alexeysol/geekregime/apiusers/configurations/JpaConfiguration.kt
package com.github.alexeysol.geekregime.apiusers.configurations

import com.github.alexeysol.geekregime.apicommons.utils.search.SearchableRepositoryImpl
import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@Configuration
@EnableJpaRepositories(
    basePackages = ["com.github.alexeysol.geekregime.apiusers.repositories"],
    repositoryBaseClass = SearchableRepositoryImpl::class
)
class JpaConfiguration
